src/include/catalog/namespace.h

```diff
@@ -35,6 +36,7 @@ typedef struct _FuncCandidateList
 	int			nvargs;			/* number of args to become variadic array */
 	int			ndargs;			/* number of defaulted args */
 	int		   *argnumbers;		/* args' positional indexes, if named call */
+	List	   *tsql_argdefaults;    /* list of default args, only set for PL/tsql function */
```

Comment:
I am hesitate to add new member here, however, I think I understand the purpose and merit.  Let's accept it for the moment.